server/handler/query.go

```diff
@@ -50,6 +51,9 @@ func genericVals(colTypes []string) []interface{} {
 // the rows as JSON
 func Query(db service.SQLDB) RequestProcessFunc {
 	return func(r *http.Request) (*serializer.Response, error) {
+		ctx, cancel := context.WithCancel(r.Context())
```

Comment:
why do you need to wrap it in WithCancel?

Reply:
It's true, we don't need the derived context, because we are not calling cancel anywhere.
I think I saw a couple of examples where they did this, but it was probably to accommodate a more generic scenario where you want to cancel early for some reason.

Should we remove it and use the request context instead?